src/components/Alert/__snapshots__/Alert.test.js.snap

```diff
@@ -20,7 +35,167 @@ exports[`Alert renders properly 1`] = `
     className="pficon pficon-error-circle-o"
   />
   <span>
-    Danger Will Robinson!
+    Alert Message!
+  </span>
+</div>
+`;
+
+exports[`Alert error renders properly 1`] = `
+<div
+  className="alert alert-danger"
+>
+  <span
+    aria-hidden="true"
+    className="pficon pficon-error-circle-o"
+  />
+  <span>
+    Alert Message!
+  </span>
+</div>
+`;
+
+exports[`Alert error renders properly with dismiss button 1`] = `
+<div
+  className="alert alert-danger alert-dismissable"
+>
+  <button
+    aria-hidden="true"
+    className="close close-default"
```

Comment:
this looks good to me...

only question is about introducing `close-default` class. should we set `bsClass` to empty string `""` and `className` to `close` on the `Button` to avoid that new class?
https://react-bootstrap.github.io/components.html#buttons-props

It doesn't seem to matter to much either way...

Reply:
@priley86 - good idea :+1:
 will update soon!

Reply:
@priley86 - It doesn't really worked  :disappointed: 
```js
// props
bsClass=""
className="close"
// result
className="close  -default"
```

Reply:
Ok - interesting they chose to do it that way. I’m fine with leaving as is. Maybe we can contribute a change upstream some day to prevent it.

Reply:
i noted that setting `bsClass=" "` (empty space) seems to work in removing the base prefix  class, but that seems like a hack and I'm not sure how browsers interpret it (works in most modern browsers it seems). Ideally there is an API level prop to remove it. I'm fine w/ either though.

@jgiardino @jeff-phillips-18 thoughts? would be nice to merge this one soon I guess.

Reply:
I'm OK with this as is, we can adjust if we feel its necessary.